docs/running-on-mesos.md

```diff
@@ -156,8 +156,10 @@ passing in the Mesos master URL (e.g: mesos://host:5050). This starts the `Mesos
 If you like to run the `MesosClusterDispatcher` with Marathon, you need to run the `MesosClusterDispatcher` in the foreground (i.e: `bin/spark-class org.apache.spark.deploy.mesos.MesosClusterDispatcher`).
 
 From the client, you can submit a job to Mesos cluster by running `spark-submit` and specifying the master URL
-to the URL of the `MesosClusterDispatcher` (e.g: mesos://dispatcher:7077). You can view driver statuses on the
-Spark cluster Web UI.
+to the URL of the `MesosClusterDispatcher` (e.g: mesos://dispatcher:7077 or mesoss://dispatcher:7077). You can view driver statuses on the
+Spark cluster Web UI. The `mesos://` scheme is used for a dispatcher that uses http REST for job submission. If
+the dispatcher is configured with `spark.ssl.enabled`, `mesoss://` is to be used.
```

Comment:
I know people hate grammar reviews, but please write in the active voice:

"Use `mesoss://` if the dispatcher is configured with `spark.ssl.enabled=true`"

Also, the preceding sentence isn't quite true, since a `mesoss://` server also uses http.  I think we can just remove it.